src/pypg/traits/observable.py

```diff
@@ -75,6 +78,8 @@ def _deliver_queue(self):
                         continue
                     self._queue_empty_event.set()
                     self.data_event.clear()
+                    self._delivery_thread = None
```

Comment:
note this is done under a lock on line 77, and the corresponding allocation is also on line 55.

Reply:
asdf